### ISSUE TITLE
Freetext import - Alternative mode: 1 line = 1 forensic artifact 

### ISSUE THREAD

**Issue by MichaelDwucet** [MISP/MISP#1740]
An idea for the freetext import tool in MISP:

In forensic reports, analysis excel sheets, etc. I often see that one line of text represents one forensic artifact.
E.g. in one line, there is some $Hashsum and one $filename or multiple $hashsums and $filename.
When I copy and paste the forensic report in the freetext import tool, the connection between the $Hashsum(s) and $filename get lost.
So just as an idea:
A second mode for the freetext import tool, that can be activated with a checkbox in the IOC import dialog. 1line = 1 artifact.
With this mode, the freetext import tool will consider all IOCs that are in one line of text, as IOCs of a single forensic artifact and combine them where possible.
E.g. 
A detected $hashsum and a $filename in one line will create a combined $filename|$hashsum attribute, instead of two distinct attributes $filename and $hashsum.

**Comment by iglocska** [comment]
Indeed, very good idea. We could build composite objects based on the same line relation as well as objects later on once we have them in MISP.
